harvard-lts/CURIOSity
# removes clear all from browse search

**removes clear all from browse search**
* * *

**JIRA Ticket**: https://github.com/harvard-lts/CURIOSity/issues/295

# What does this Pull Request do?
A fix for ticket https://github.com/harvard-lts/CURIOSity/issues/295, removes the clear all button when the page is a browse page.

# How should this be tested?

Spin up this branch, and go to the demo exhibit Browse. Pick a category, and then run a search in the secondary search box in that category. It should work and you will not see the clear all button

Then to confirm you still get the clear all in a regular search, perform a regular search. The button should be there.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
